spec/02-integration/04-admin_api/15-off_spec.lua

```diff
@@ -367,6 +367,50 @@ describe("Admin API #off", function()
         assert.response(res).has.status(201)
       end)
 
+      it("hides workspace related fields from /config response", function()
+        local res = assert(client:send {
+          method = "POST",
+          path = "/config",
+          body = {
+            config = [[
+            _format_version: "1.1"
+
+            services:
+            - name: my-service
+              url: https://example.com
+              plugins:
+              - name: file-log
+                config:
+                  path: /tmp/file.log
+              - name: key-auth
+              routes:
+              - name: my-route
+                paths:
+                - /
+
+            consumers:
+            - username: my-user
+              keyauth_credentials:
+              - key: my-key
+            ]],
+          },
+          headers = {
+            ["Content-Type"] = "application/json"
+          }
+        })
+
+        local body = assert.response(res).has.status(201)
+        local entities = cjson.decode(body)
+
+        assert.is_nil(entities.workspaces)
+        assert.is_nil(entities.consumers["4b1b701d-de2b-5588-9aa2-3b97061d9f52"].ws_id)
+        assert.is_nil(entities.keyauth_credentials["487ab43c-b2c9-51ec-8da5-367586ea2b61"].ws_id)
+        assert.is_nil(entities.plugins["0611a5a9-de73-5a2d-a4e6-6a38ad4c3cb2"].ws_id)
+        assert.is_nil(entities.plugins["661199ff-aa1c-5498-982c-d57a4bd6e48b"].ws_id)
+        assert.is_nil(entities.routes["481a9539-f49c-51b6-b2e2-fe99ee68866c"].ws_id)
+        assert.is_nil(entities.services["0855b320-0dd2-547d-891d-601e9b38647f"].ws_id)
```

Comment:
Are these uuids deterministic across test runs, even if we run tests in isolation?

Reply:
You can ensure determinism by adding the `id:` entries to the yaml above

Reply:
Yes. In fact if they weren't the call will cause Lua errors of referencing a field on `nil` value :)